cpp/foxglove/src/server.cpp

```diff
@@ -257,6 +257,13 @@ FoxgloveResult<WebSocketServer> WebSocketServer::create(
     };
   }
 
+  if (options.tls_identity) {
+    c_options.tls_cert = reinterpret_cast<const uint8_t*>(options.tls_identity->cert.data());
```

Comment:
A reinterpret cast from byte to uint8_t? Something is odd about that. Maybe use the latter type in the vector instead of byte?

Strange C++ doesn't realize they're the same

Reply:
Yeah ahaha this has been a fun thing to work through on C++, they're not the same type in C++'s eyes and implicit casting between them is specifically disallowed. I also don't love that this proliferates, but it's everywhere in bridge code (for example)